nautobot/dcim/navigation.py

```diff
@@ -329,6 +329,22 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
+                        link="dcim:interfaceredundancygroup_list",
+                        name="Interface Redundancy Groups",
+                        weight=600,
+                        permissions=[
+                            "dcim.view_interfaceredundancygroup",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
+                                link="dcim:interfaceredundancygroup_add",
+                                permissions=[
+                                    "dcim.add_interfaceredundancygroup",
+                                ],
+                            ),
+                        ),
+                    ),
```

Comment:
For some reason the `add` and `import` routes are not being added. I'm getting this error in the UI:

<img width="274" alt="image" src="https://github.com/nautobot/nautobot/assets/138052/c7541ccc-82cf-4f4a-be14-50171e431c65">

And confirmed that the route does not exist for `add`: 

```python
In [11]: reverse("dcim:interfaceredundancygroup_add")
...
NoReverseMatch: Reverse for 'interfaceredundancygroup_add' not found. 'interfaceredundancygroup_add' is not a valid view function or pattern name.
```

Additionally, it looks like it might be a bug in the `NautobotUIViewSetRouter` code, seeing as there's no `_add` or `_import` route names being generated, but all the others are: 

```python
In [17]: from nautobot.dcim import urls

In [18]: [u for u in urls.urlpatterns if u.name.startswith("interfacered")]
Out[18]:
[<URLPattern '^interface-redundancy-groups/$' [name='interfaceredundancygroup_list']>,
 <URLPattern '^interface-redundancy-groups/edit/$' [name='interfaceredundancygroup_bulk_edit']>,
 <URLPattern '^interface-redundancy-groups/delete/$' [name='interfaceredundancygroup_bulk_delete']>,
 <URLPattern '^interface-redundancy-groups/(?P<pk>[^/.]+)/$' [name='interfaceredundancygroup']>,
 <URLPattern '^interface-redundancy-groups/(?P<pk>[^/.]+)/delete/$' [name='interfaceredundancygroup_delete']>]
```

Reply:
Ah nevermind I see tha the very views doesn't implement create or bulk_create...

```python
class InterfaceRedundancyGroupUIViewSet(
    ObjectDetailViewMixin,
    ObjectListViewMixin,
    ObjectDestroyViewMixin,
    ObjectBulkDestroyViewMixin,
    ObjectBulkUpdateViewMixin,
):
```

Reply:
Which begs the question: Are we literally just trying to cherry-pick or are we going for feature-completeness in this PR?

Reply:
@grelleum Since you're working on this FYI 😁